src/main/views/MattermostBrowserView.ts

```diff
@@ -175,6 +192,7 @@ export class MattermostBrowserView extends EventEmitter {
         } else {
             loadURL = this.view.url.toString();
         }
+        AppState.updateExpired(this.id, false);
```

Comment:
Naming is hard :P
`AppState` actually refers to the storage place we use for the number mention. unread flag, and the session expired flag. This is all here because it affects the badges shown on the tabs.

The `expired` map shows which tabs are marked as "session expired" so that we will show the little `(!)` badge.